src/qt/test/rpcnestedtests.cpp

```diff
@@ -46,6 +46,9 @@ void RPCNestedTests::rpcNestedTests()
         tableRPC.appendCommand(c.name, &c);
     }
 
+    // Call ECC_Stop before instantiating another NodeContext in TestingSetup
```

Comment:
This is really unfortunate, but I can't think of anything less hack-ish. And at least it's relegated to the test code.